game_event_player/migrations/0002_rename_game_event_id_gameeventplayer_game_event_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2 on 2023-04-30 13:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
new migration should also deal with unique together constraints update, you can have some reference from here #58

Reply:
> new migration should also deal with unique together constraints update, you can have some reference from here #58

Hi, Thank you I did it

Reply:
sorry I didn't mention it before, it should be auto-generated by Django, you should not make the changes manually.
was it auto-generated here?

Reply:
> sorry I didn't mention it before, it should be auto-generated by Django, you should not make the changes manually. was it auto-generated here?

Yes, I did the make migration command for it